### DISCUSSION TITLE
In which bucket would SurrealDB fit best?

### DISCUSSION THREAD

**Discussion by RaphaelTarita** [ScoopInstaller/Scoop#5347] [Help and Questions] [answered]
I'd like to propose adding a manifest for [SurrealDB](https://surrealdb.com/), but I don't know in which bucket repository I should open up an issue for that. I personally know nothing about writing/adding manifests to scoop, but I found out that there is a manifest in a community bucket (https://github.com/AntonOks/scoop-aoks/blob/master/bucket/surrealdb-aoks.json) via ScoopSearch.

Which bucket do you think this would fit in best, or in general, where should I open up the issue?

**Comment by rashil2000** [comment, marked as answer]
The Main bucket is appropriate for SurrealDB
